tasks/security_scorecard/security_scorecard.rb

```diff
@@ -61,7 +61,7 @@ def ssc_issue_to_kdi_asset_hash(issue)
         ###
         if issue["connection_attributes"]
           if issue["connection_attributes"].is_a? Hash
-            port = issue["connection_attributes"]["dst_port"]
+            # port = issue["connection_attributes"]["dst_port"]
```

Comment:
is this what it looks like? that the line is commented out?

Reply:
Yes,  it was set but not called anywhere else.  I just didn't delete the lines in case they were needed later.